raysect_mayavi/primitives/geometric/utility.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+
+from math import sqrt
+
+from scipy.spatial import Delaunay
+
```

Comment:
Looks good, it may be worth us refactoring some of my other code to use these utility methods.

Reply:
Scratch that, I see you've already done that in some other files :)